### PR TITLE
Improve the debug message.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: python/lsst/ts/m2/csc.py [hunk 1]
```diff
@@ -525,17 +545,25 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
                     retry_times=self.ilc_retry_times,  # type: ignore[arg-type]
                 )
 
-            except RuntimeError:
-                # TODO: Publish the list of failed ILCs after the DM-40146 is
-                # done.
+            except RuntimeError as error:
                 await self._basic_cleanup_and_power_off_motor()
+
+                error.add_note("Power-off the motor.")
```
Comment (tribeiro) [inline python/lsst/ts/m2/csc.py:551]: Not sure I understand what is goin on here. Can you clarify a bit?

If I understood it correctly you check if the ILC modes are enabled and if it is not you try to enable them. If that fails you power off the motors. If this is correct, maybe expand the error message to something like:

```
Failed to enable ILCs. Powering off motors.
```

would be good if you could add some suggestion of what could be done to work around or troubleshoot the problem.

File: python/lsst/ts/m2/csc.py [hunk 2]
```diff
@@ -525,17 +545,25 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
...
+            await self._execute_command(
+                self.controller_cell.set_closed_loop_control_mode,
+                MTM2.ClosedLoopControlMode.OpenLoop,
+                timeout=self.COMMAND_TIMEOUT_LONG,
+            )
+
+        except RuntimeError as error:
+            if self._exists_error_in_controller():
+                error.add_note(
+                    "The existed error blocks the transition to open-loop control."
```
Comment (tribeiro) [inline python/lsst/ts/m2/csc.py:564]: Consider rephrasing:

```
Cannot transition to open-loop control due to existing errors.
```

Also, is there any hint on what people can/should do?

**Review by tribeiro — COMMENTED** [review]
Can you please take a look at my comments? I would like to request some level of verbosity to the operation if you don't mind! thanks! 

I would like to take another look before you merge so, let me know!

File: python/lsst/ts/m2/csc.py [hunk 3]
```diff
@@ -464,6 +474,10 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
 
         await self._bypass_error_codes()
 
+        # Check there is any existed error or not
+        if self._exists_error_in_controller():
+            raise RuntimeError("Error exists. Try to clear the error first.")
```
[the remark this answers is not in the document]
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py]: Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!

File: python/lsst/ts/m2/csc.py [hunk 4]
```diff
@@ -510,12 +524,18 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
             True,
             timeout=self.COMMAND_TIMEOUT_LONG,
         )
-        await self._execute_command(
-            self.controller_cell.power,
-            MTM2.PowerType.Motor,
-            True,
-            timeout=self.COMMAND_TIMEOUT_LONG,
-        )
+
```
[the remark this answers is not in the document]
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py:545]: Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!

File: python/lsst/ts/m2/csc.py [hunk 5]
```diff
@@ -510,12 +524,18 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
...
+        try:
+            await self._execute_command(
+                self.controller_cell.power,
+                MTM2.PowerType.Motor,
+                True,
+                timeout=self.COMMAND_TIMEOUT_LONG,
+            )
+
+        except RuntimeError as error:
+            error.add_note("Please check/reset the interlock or power system.")
```
[the remark this answers is not in the document]
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py]: Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!

File: python/lsst/ts/m2/csc.py (hunk 1 above)
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py]: Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!

File: python/lsst/ts/m2/csc.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py]: Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!

**Review by tribeiro — APPROVED** [review]
Amazing! thank you so much!

**Pull request merged**